docs/core/diagnostics/debug-highcpu.md

```diff
@@ -96,17 +96,35 @@ Press p to pause, r to resume, q to quit.
     Status: Running
 
 [System.Runtime]
+    % Time in GC since last GC (%)                         0
+    Allocation Rate / 1 sec (B)                            0
     CPU Usage (%)                                         25
+    Exception Count / 1 sec                                0
+    GC Heap Size (MB)                                      4
+    Gen 0 GC Count / 60 sec                                0
+    Gen 0 Size (B)                                         0
+    Gen 1 GC Count / 60 sec                                0
+    Gen 1 Size (B)                                         0
+    Gen 2 GC Count / 60 sec                                0
+    Gen 2 Size (B)                                         0
+    LOH Size (B)                                           0
+    Monitor Lock Contention Count / 1 sec                  0
+    Number of Active Timers                                1
+    Number of Assemblies Loaded                          140
+    ThreadPool Completed Work Item Count / 1 sec           3
+    ThreadPool Queue Length                                0
+    ThreadPool Thread Count                                7
+    Working Set (MB)                                      63
 ```
 
 Throughout the duration of the request, the CPU usage will hover around 25% . Depending on the host machine, expect varying CPU usage.
 
 > [!TIP]
 > To visualize an even higher CPU usage, you can exercise this endpoint in multiple browser tabs simultaneously.
 
-At this point, you can safely say the CPU is running higher than you expect.
+At this point, you can safely say the CPU is running higher than you expect, meaning the app has a slow request. 
```

Comment:
> meaning the app has a slow request.

There can be a myriad of reasons for higher cpu. Maybe write it as something like
```suggestion
At this point, you can safely say the CPU is running higher than you expect. With the knowledge of the app-code, and the requests made this means the app has a slow request. 
```
?